src/test/Components/Blueprints/ImportBlueprintModal.test.tsx

```diff
@@ -555,9 +555,6 @@ describe('Import modal', () => {
         await screen.findByPlaceholderText('Paste your public SSH key')
       )
     );
-    expect(
```

Comment:
The same here, I've tried running the tests with this expect and the groups validation and they seem to be passing just fine, I'd keep this in to keep some validation even if not perfect :shrug: